src/wagtail_personalisation/wagtail_hooks.py

```diff
@@ -129,7 +129,7 @@ def dont_show_variant(parent_page, pages, request):
 if WAGTAIL_VERSION >= (5, 2):
 
     @hooks.register("register_page_listing_buttons")
-    def page_listing_variant_buttons(page, user, *args, **is_parent):
+    def page_listing_variant_buttons(page, user, next_url, *args, **is_parent):
```

Comment:
:+1: Change required by 5.2 as described [here](https://docs.wagtail.org/en/latest/releases/5.2.html#hooks-for-page-listing-and-page-header-buttons-no-longer-accept-a-page-perms-argument)